src/components/radio-icon/radio-box.tsx

```diff
@@ -75,7 +78,10 @@ export default function RadioBox({
       >
         {description}
       </ThemeText>
-      <View style={styles.radioIcon}>
+      <View
+        style={styles.radioIcon}
+        testID={selected ? 'radioChecked' : 'radioNotChecked'}
```

Comment:
Er tanken at IDer skal være unike her? Vil i så fall dette bli et problem når der er flere enn ett element med ID "radioNotChecked"?

Reply:
Hm. Du har rett. Bra catch! Jeg tenkte ikke veldig langt for akkurat nå benyttes denne testID-en kun i valget mellom t:kort og mobiltoken i token toggle - ergo, er den ene cheked, mens den andre er unchecked. Mulig jeg kunne sendt inn konteksten fra hvor den benyttes for å gjøre den entydig.

Men samtidig kan jeg gjøre det når jeg trenger denne andre steder. Skal se på det.

Reply:
Ah, forstår. Er det en ok løsning å slå det sammen med `testID`? Så kan man håndtere at de er unike på nivået over.

```suggestion
        testID={testID + (selected ? 'radioChecked' : 'radioNotChecked')}
```

Reply:
Ser at per nå benyttes denne komponenten kun fra SelectTravelTokenScreen, så akkurat nå er denne grei nok. Ved behov så kan testene f.ex først finne elementet rundt (testID for TouchableOpacity) og deretter det aktuelle elementet (testID for radioChecked). 🤷

Reply:
Mm. Tenkte på den også. Jeg kan ta inn endringen her hos meg, for jeg må samtidig endre testene til å bruke denne id-en